### PR TITLE
Removed Mocha, Chai in favor of Jest.  src/test -> __tests__. added n…

### PR DESCRIPTION
**Pull request by jwineman**
…pm run clean, npm run cover.

### CODE REVIEW

File: src/test/reducers/zonePurgeCacheTest.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
-import { zonePurgeCacheReducer } from '../../src/reducers/zonePurgeCache';
-import * as ActionTypes from '../../src/constants/ActionTypes';
+import { zonePurgeCacheReducer } from '../..//reducers/zonePurgeCache';
+import * as ActionTypes from '../..//constants/ActionTypes';
```
Comment (tajo) [inline src/test/reducers/zonePurgeCacheTest.js]: find & replace that went wrong?

File: src/test/utils/PluginAPITest.js [hunk 2]
```diff
@@ -9,6 +8,6 @@ describe('PluginAPI', () => {
           success: true
         }
       })
-    ).to.eql(true);
+    ).toEqual(true);
```
Comment (tajo) [inline src/test/utils/PluginAPITest.js]: use toBeTruthy()

File: src/test/selectors/configTest.js [hunk 3]
```diff
@@ -53,6 +52,6 @@ describe('Config Selector', () => {
         },
         'isDNSPageEnabled'
       )
-    ).to.eql(true);
+    ).toEqual(true);
```
Comment (tajo) [inline src/test/selectors/configTest.js]: use toBeTruthy()

File: src/test/selectors/configTest.js [hunk 4]
```diff
@@ -40,7 +39,7 @@ describe('Config Selector', () => {
         },
         'key'
       )
-    ).to.eql('value');
+    ).toEqual('value');
```
Comment (tajo) [inline src/test/selectors/configTest.js]: use toBe()

File: src/test/selectors/configTest.js [hunk 5]
```diff
@@ -26,7 +25,7 @@ describe('Config Selector', () => {
           },
           'test.html'
         )
-      ).to.eql('test.html');
+      ).toEqual('test.html');
```
Comment (tajo) [inline src/test/selectors/configTest.js]: use toBe()

File: src/test/selectors/configTest.js [hunk 6]
```diff
@@ -13,7 +12,7 @@ describe('Config Selector', () => {
         },
         'test.html'
       )
-    ).to.eql('http://site.com/test.html');
+    ).toEqual('http://site.com/test.html');
```
Comment (tajo) [inline src/test/selectors/configTest.js]: use toBe()

File: package.json [hunk 7]
```diff
@@ -6,9 +6,18 @@
   "scripts": {
     "build": "NODE_ENV=development node_modules/webpack/bin/webpack.js --progress",
     "build:production": "NODE_ENV=production node_modules/webpack/bin/webpack.js --progress",
-    "lint": "node_modules/eslint/bin/eslint.js src/ --ext js --ext json",
+    "clean": "rm -rf .jest && rm -rf node_modules",
+    "cover": "node_modules/jest/bin/jest.js --coverage",
     "format": "node_modules/eslint/bin/eslint.js src/ --fix --ext js --ext json",
-    "test": "mocha src/test/*/*.js --compilers js:babel-register"
+    "lint": "node_modules/eslint/bin/eslint.js src/ --ext js --ext json",
+    "test": "node_modules/jest/bin/jest.js"
```
Comment (tajo) [inline package.json]: why not just 

```sh
"test": "jest"
```

The same for eslint. Afaik, npm does the look up in local node_modules automatically.

Comment (jwineman) [inline package.json]: I didn't realize that, I always thought that was relying on global install.

**Pull request merged**